docs/source/guide/clv/clv_intro.md

```diff
@@ -0,0 +1,48 @@
+# Introduction to Customer Lifetime Value
+
+For a variety of reasons, companies often need quantify the monetary value of each customer (present and future) and do so while acknowledging the entire relationship the customer has with the company.  This value represents the maximum the company should spend on acquiring new customers.  It may also guide customer retention efforts and inform financial forecasting.
```

Comment:
```suggestion
For a variety of reasons, companies often need to quantify the monetary value of each customer (present and future) and do so while acknowledging the entire relationship the customer has with the company.  This value represents the maximum the company should spend on acquiring new customers.  It may also guide customer retention efforts and inform financial forecasting.
```